shell/pages/c/_cluster/manager/drivers/kontainerDriver/_id.vue

```diff
@@ -0,0 +1,15 @@
+<script>
+import ResourceDetail from '@shell/components/ResourceDetail';
+
+export default {
+  name:       'KontainerDriverEditPage',
+  components: { ResourceDetail },
+};
+</script>
+
+<template>
+  <ResourceDetail
+    resource-override="kontainerDriver"
```

Comment:
why do we use `resource-override` and `parent-route-override`?

Reply:
Because cloudCredential uses it, so I assumed it is needed for something I don't fully understand. which you are right, it doesn't in this case